packages/core/src/tools/serialisation/sanitize.spec.ts

```diff
@@ -85,6 +85,23 @@ describe('sanitize', () => {
       })
     })
 
+    it('should serialize errors as JSON.stringify does', () => {
+      // Explicitely keep the previous behavior to avoid breaking changes in 4.x
+      // Browsers have different behaviors:
+      // IE11 adds a description field
+      // Safari IOS12 adds parts of the stack
+      const error = new Error('My Error')
+      expect(JSON.stringify(sanitize(error))).toEqual(JSON.stringify(error))
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏did you try using `{ ...error }` or `Object.assign({}, error)` instead of `JSON.stringify`? It should yield better message when the assertion fails
 
```suggestion
      expect(sanitize(error)).toEqual({ ...error })
```